app/controllers/submissions_controller.rb

```diff
@@ -0,0 +1,28 @@
+# Handle user submissions of soundbites
+class SubmissionsController < ApplicationController
+  def new
+    @submission = Submission.new
+  end
+
+  def create
+    @submission = Submission.new submission_params
+    if @submission.valid?
+      if params[:do_not_use].blank?
+        @submission.save
+        flash[:success] = 'Thank you for your submission!'
+        redirect_to submit_path
+        NotificationMailer.submission(soundbite: @submission).deliver_now
+      end
+    else
+      render 'new'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.